elasticsearch_collectd.py

```diff
@@ -517,6 +532,9 @@ def init_stats():
                   "thread_pool"
     NODE_STATS_CUR = dict(NODE_STATS.items())
     INDEX_STATS_CUR = dict(INDEX_STATS.items())
+    if ES_VERSION.startswith("2."):
+        NODE_STATS_CUR.update(NODE_STATS_ES_2)
+
     if ES_VERSION.startswith("1.1") or ES_VERSION.startswith("1.2"):
```

Comment:
There's an ES_VERSION not None in that method so if you don't specify in the plugin config, that will be the default.